src/Messages/Strings.hs

```diff
@@ -49,6 +49,8 @@ showErrorMessage OutputAndValidate =
 showErrorMessage (MustSpecifyVersionWithUpgrade elmVersion) =
     "I can only upgrade code to the latest Elm version.  To make sure I'm doing what you expect, you must also specify --elm-version=" ++ show elmVersion ++ " when you use --upgrade."
 
+showErrorMessage NoInputs =
+    "No inputs"
```

Comment:
Why did this not lead to crashes before?  Is `NoInputs` actually used anywhere?

Reply:
src/ElmFormat.hs:
```
                    case (elmVersionResult, determineWhatToDoFromConfig config resolvedInputFiles) of
                        (_, Left NoInputs) ->
                            (handleParseResult $ Flags.showHelpText defaultVersion elmFormatVersion experimental)
                                >> exitFailure

                        (_, Left message) ->
                            exitWithError message
...
```

`exitWithError` uses `showErrorMessage` (which is what the warning is about). So the `NoInputs` case is effectively impossible because it is handled in a special way.